wireapp/wire-android
# feat: media files tab - Part 1 (WPB-5378)

<table>
<td>
  <a href="https://wearezeta.atlassian.net/browse/WPB-5378" title="WPB-5378" target="_blank"><img alt="Task" src="https://wearezeta.atlassian.net/rest/api/2/universal_avatar/view/type/issuetype/avatar/10818?size=medium" />WPB-5378</a>  Files tab in media gallery
  </td></table>
  <br />
 




----
#### PR Submission Checklist for internal contributors

- The **PR Title**

- The **PR Description**
----

# What's new in this PR?

### Issues

There was no empty states and no tabs for (PICTURES, FILES)

### Causes (Optional)

Not implemented

### Solutions

Add tabs with (PICTURES, FILES) and empty state for both screens

### Dependencies (Optional)

### Testing

#### How to Test

- Open Conversation Details
- Open Media Screen
- Verify that there are tabs and there are empty states for both tabs

### Notes (Optional)

This is the first of many PRs to follow along for the implementation of displaying files.

### Attachments (Optional)

<img src="https://github.com/wireapp/wire-android/assets/5890660/fd984783-6290-4d9d-8326-fdaf512a2ae2" width="200" height="400" />

<img src="https://github.com/wireapp/wire-android/assets/5890660/8d788186-45b8-4698-89ae-5249f0d00bb9" width="200" height="400" />
